docs/reference-guide/04-active_windows.md

```diff
@@ -135,6 +135,8 @@ A built-in notepad for recording comments regarding the study. Such comments typ
 
 ## Load
 
+_[Documentation of the GUI](https://antares-web.readthedocs.io/en/latest/user-guide/study/areas/02-load/)_
+
```

Comment:
Je pense qu'il faudrait dès le début de cette page

CETTE IHM EST DÉPRÉCIÉE, merci d'utiliser plutôt Antares Web => lien vers la doc

Aussi "GUI" est très vague, je pense qu'il vaut mieux parler d'Antares Web pour éviter toute confusion.

Reply:
J'ai ajouté un disclaimer en début de partie (qui sera à enlever une fois la doc refondue) :
**DISCLAIMER: this section sometimes refers to the original interfaces of Antares-Simulator, which are deprecated. Please refer to the AntaresWeb online documentation for the description of the current interface.**

Reply:
J'ai aussi fait référence aux "interfaces AntaresWeb" plutôt qu'au GUI dans les différentes sections

Reply:
En ce qui concerne les onglets (commentaire d'en-dessous), ils étaient déjà décrits dans la doc AntaresWeb : j'ai donc été un peu plus générique dans la doc Simulator, en gardant l'explication sur leur sens métier (qui reste à mettre ici), mais en étant un peu moins spécifique sur les "tabs". Ceci dit, je pense que l'explication métier a toujours du sens ici.